content/300-guides/125-development-environment/050-environment-variables/index.mdx

```diff
@@ -77,14 +77,30 @@ Refer to the `dotenv` documentation for information about [what happens if an en
 
 ### Expanding variables
 
-Variables stored in `.env` files can be expanded using the format specified by [dotenv-expand](https://github.com/motdotla/dotenv-expand). This is useful when dealing with PaaS or similar products. such as Heroku):
+Variables stored in `.env` files can be expanded using the format specified by [dotenv-expand](https://github.com/motdotla/dotenv-expand). 
 
 ```
 # .env
 DATABASE_URL=postgresql://test:test@localhost:5432/test
 DATABASE_URL_WITH_SCHEMA=${DATABASE_URL}?schema=public
 ```
 
+This will make the environment variable `DATABASE_URL_WITH_SCHEMA` with value `postgresql://test:test@localhost:5432/test?schema=public` available for Prisma.
+
+You can also use environment variables in the expansion that are set _outside_ of the `.env` file, for example a database URL that is set on a PaaS like Heroku or similar:
+
+```terminal
+# environment variable already set in the environment of the system
+export DATABASE_URL=postgresql://test:test@localhost:5432/test
+```
```

Comment:
````suggestion
```env file=.env
# environment variable already set in the environment of the system
export DATABASE_URL=postgresql://test:test@localhost:5432/test
```
````

Reply:
No, this is actually a command you execute in the terminal to set the env var